src/dbnode/ts/downsampled/counter_resets_downsampler.go

```diff
@@ -18,14 +18,14 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package tile
+package downsampled
```

Comment:
nit: `downsample`

Reply:
Tried that initially, but then I didn't like the `downsample.DownsampledValue`, decided to go for `downsampled.Value` instead. With that in mind, what do you think?

Reply:
IMO `downsample.Value` is a happy middle ground but don't have particularly strong feelings either way here :)